CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.6.1
```

Comment:
`2.6.0` is not released yet, this could be added to that

Reply:
see https://github.com/Sage/carbon/releases for latest versions

Reply:
okay, will rebase this commit to move changes to `2.6.0`